python/lib/dependabot/python/update_checker/index_finder.rb

```diff
@@ -120,6 +120,8 @@ def pyproject_index_urls
           sources.each do |source|
             if source["default"]
               urls[:main] = source["url"]
+            elsif source["name"] == "PyPI"
+              urls[:main] = "https://pypi.org/pypi/"
```

Comment:
Is this the correct index to target? Or does it need to point to `/simple`?

Reply:
I copied that url from https://github.com/dependabot/dependabot-core/blob/97fc09053fc9fe5e6a14192bffe3055cefc9db20/python/lib/dependabot/python/metadata_finder.rb#L15 and just figured that should work, but I'm not sure.

It seems poetry doesn't require the `url` to be there at all (assuming that since the change introduced by running `poetry source add pypi` doesn't add a `url`), so in some sense I'm simply trying to get dependabot to not crash here, and assuming that poetry itself will work as it does when running it locally 😄

Reply:
According to poetry source, it does `https://pypi.org/simple`:

https://github.com/python-poetry/poetry/blob/01657ddeca182259e4350794a4f378241336085f/src/poetry/repositories/pypi_repository.py#L35-L47

Reply:
Sounds good to me, @maksbotan 👍🏼 I changed it now